Dockerfile/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.8-slim-buster
+
+RUN pip install --upgrade pip
+
+RUN apt-get update
+
+RUN apt-get install curl unzip --yes
+
+RUN apt-get update && apt-get install -y nodejs npm
```

Comment:
Why do you install NodeJS twice first in line 9 and again with a another way in line 10?

Reply:
Solved this issue locally, will update PR once rest of the issues are resolved